python/neuroglancer/pipeline/task_queue.py

```diff
@@ -172,7 +179,7 @@ def insert(self, task):
         }
 
         def cloud_insertion(api):
-            api.insert(
+            api.tasks().insert(
```

Comment:
why did you decide to move tasks() here? 
when you call tasks is it when the document describing the api get downloads and the methods created based on it?